src/app/MainMenu.js

```diff
@@ -107,7 +108,7 @@ export const MainMenu = (activePage) => {
 
```

Comment:
Your have to add class or ids to your elements in searching section and then refer to them by it, because you override buttons and inputs from my feature (see the video i sent on discord in  'radosław-bajor-projekt-2'